### PR TITLE
Update docs home page

### PR DESCRIPTION
**Pull request by willingc**
- Add visual interest above the fold with a chart image
- Reorder sections and drop their heading levels
- Add a note about where to find 1.x documentation

### CODE REVIEW

File: docs/public/assets/markdown/home.md [hunk 1]
```diff
@@ -1,45 +1,61 @@
...
 
 ```js
 import NetworkFrame from "semiotic/lib/NetworkFrame"
 ```
 
-or you can use the bundled version on unpkg.com
+Alternatively, you can use semiotic's bundled version available on unpkg.com.
 
 ```html
 <script src="https://unpkg.com/semiotic" />
```
Comment (alexeyraspopov) [inline docs/public/assets/markdown/home.md]: This part also needs an update, the same as in root README

```suggestion
<script type="module">
  import { XYFrame } from "https://unpkg.com/semiotic?module";
</script>
```

**Review by alexeyraspopov — APPROVED** [review]
I guess we still planning to merge package.json and README so docs will be just a bunch of content files, right?

**Comment by willingc** [comment]
> I guess we still planning to merge package.json and README so docs will be just a bunch of content files, right?

@alexeyraspopov That would be my preference. Right now the docs are pretty awkward to author.

**Comment by willingc** [comment]
Thanks for the review too.

**Pull request merged**